Typography/index.css

```diff
@@ -0,0 +1,91 @@
+@import url("../main.css");
+@import url("../utility.css");
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  background-color: var(--darl-grey);
+  color: var(--white);
+}
+
+/* Typography Styles: Start */
+h1 {
+  font-size: 3rem;
+  line-height: 4rem;
+  font-weight: 600;
```

Comment:
As you've created a `text-600` class for `font-weight: 600`, then it could be avoided in h1, h2...h5. You can directly use the class of `text-600` for h1, h2...h5. Just a suggestion to avoid code repetition.